test/ios-entitlements-service.ts

```diff
@@ -0,0 +1,173 @@
+import temp = require("temp");
+import { EOL } from "os";
+import { assert } from "chai";
+import { IOSEntitlementsService } from "../lib/services/ios-entitlements-service";
+import * as yok from "../lib/common/yok";
+import * as stubs from "./stubs";
+import * as FsLib from "../lib/common/file-system";
+import * as path from "path";
+
+// start tracking temporary folders/files
+temp.track();
+
+describe("IOSEntitlements Service Tests", () => {
+	const createTestInjector = (): IInjector => {
+		const testInjector = new yok.Yok();
+
+		testInjector.register('platformsData', stubs.PlatformsDataStub);
+		testInjector.register("logger", stubs.LoggerStub);
+		testInjector.register('iOSEntitlementsService', IOSEntitlementsService);
+
+		testInjector.register("fs", FsLib.FileSystem);
+
+		testInjector.register("pluginsService", {
+			getAllInstalledPlugins: async () => []
+		});
+
+		return testInjector;
+	};
+
+	let injector: IInjector,
+		platformsData: any,
+		projectData: IProjectData,
+		fs: IFileSystem,
+		iOSEntitlementsService: IOSEntitlementsService,
+		destinationFilePath: string;
+
+	beforeEach(function() {
+		injector = createTestInjector();
+
+		platformsData = injector.resolve("platformsData");
+		projectData = <IProjectData>platformsData.getPlatformData();
+		projectData.projectName = 'testApp';
+
+		projectData.platformsDir = temp.mkdirSync("platformsDir");
+		projectData.projectDir = temp.mkdirSync("projectDir");
+
+		fs = injector.resolve("$fs");
+
+		iOSEntitlementsService = injector.resolve("iOSEntitlementsService");
+		destinationFilePath = iOSEntitlementsService.getPlatformsEntitlementsPath(projectData);
+	});
+
+	describe("Ensure paths constructed are correct", () => {
+		it("Ensure destination entitlements relative path is calculated correctly.", () => {
+			const expected = "testApp/testApp.entitlements";
+			let actual = iOSEntitlementsService.getPlatformsEntitlementsRelativePath(projectData);
+			assert.equal(actual, expected);
```

Comment:
That's a single string (see `expected` value). Do we need deepEqual in this case?